.github/workflows/build.yml

```diff
@@ -67,3 +68,6 @@ jobs:
           -e "KBC_STORAGE_TOKEN=${{ secrets.KBC_STORAGE_TOKEN }}" \
           -e "KBC_MANAGE_TOKEN=${{ secrets.KBC_MANAGE_TOKEN }}" \
           keboola/sandboxes-api-php-client composer test
+      - name: Dump docker logs on failure
+        if: failure()
+        uses: jwalton/gh-docker-logs@v1
```

Comment:
oh, interesting!

Reply:
Yes, works great! It helped me to finally find out why the builds were still failing on CI.
![image](https://user-images.githubusercontent.com/215660/94020837-a8221280-fdb3-11ea-9be7-52a505e0b162.png)
I had a typo in the PROVIDER env var. 😄